src/sentry/receivers/onboarding.py

```diff
@@ -574,3 +575,42 @@ def record_integration_added(
                     task=task_mapping[integration_type],
                     status=OnboardingTaskStatus.COMPLETE,
                 )
+
+
+@project_transferred.connect(weak=False, dispatch_uid="onboarding.record_project_transferred")
+def record_project_transferred(old_org_id, updated_project, **kwargs):
+
+    new_organization_id = updated_project.organization_id
+
+    analytics.record(
+        "project.transferred",
+        old_organization_id=old_org_id,
+        new_organization_id=new_organization_id,
+        project_id=updated_project.id,
+        platform=updated_project.platform,
+    )
+
+    existing_tasks_in_old_org = OrganizationOnboardingTask.objects.filter(
+        organization_id=old_org_id,
+        task__in=OrganizationOnboardingTask.TRANSFERABLE_TASKS,
+    )
+
+    existing_tasks_in_new_org = set(
+        OrganizationOnboardingTask.objects.filter(organization_id=new_organization_id).values_list(
+            "task", flat=True
+        )
+    )
+
+    tasks_to_be_transferred = []
+
+    for task in existing_tasks_in_old_org:
+        if task.task not in existing_tasks_in_new_org:
+            task_dict = {
+                key: value for key, value in task.__dict__.items() if key not in ["id", "_state"]
+            }
```

Comment:
I'm not completely satisfied with this solution and am wondering if there’s a better approach. The reason I want to avoid manually adding each field is that if we add a new field in the future, I’d have to come back here and update the code again.

Reply:
but for now I need to remove the `id` and the `state` was removed because the types complained